src/responder/common/responder_common.c

```diff
@@ -122,11 +125,26 @@ static errno_t get_client_cred(struct cli_ctx *cctx)
         return ENOMSG;
     }
 
+    if (cctx->creds->ucred.pid > -1) {
```

Comment:
Is this portable for systems that do not have /proc or where /proc is not readable?

Reply:
I believe on such a system `open(proc_path, O_RDONLY)` will fail and debug message will miss `cmd_line`, so no harm from my point of view.

So this patch improves debug message on vast majority of platforms and doesn't change anything for the rest.